src/app/components/progress-indicator/index.html

```diff
@@ -0,0 +1,253 @@
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="Regress"
+    >
+    Indicates that progress should return to the previous item. This moves from the currently active item to the item that precedes it.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="Reset"
+    >
+    Indicates that progress is incomplete. This marks all items as incomplete and sets the first progress item as the active item.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="Progress indicator item properties">
+    <sky-demo-page-property
+      propertyName="title"
+    >
+      Specifies the title to display with the progress item.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="Progress indicator reset button properties">
+    <sky-demo-page-property
+      propertyName="disabled"
+      isOptional="true"
+      defaultValue="false"
+    >
+      Indicates whether to disable the reset button.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="progressIndicator"
+    >
+      Specifies the progress indicator component to associate with the reset button.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="Progress indicator reset button events">
+    <sky-demo-page-property
+      propertyName="resetClick"
+    >
+      Fires when users select the reset button that marks all items as incomplete and sets the first progress item as the active item.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="Progress indicator nav button properties">
+    <p>
+      The progress indicator nav button component allows users to navigate the steps in modal wizards. We recommend against using it in waterfall pages and passive progress indicators.
+    </p>
+    <sky-demo-page-property
+      propertyName="buttonText"
+      isOptional="true"
+      defaultValue="Next"
+    >
+      Specifies the label to display on the nav button.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="buttonType"
+      isOptional="true"
+      defaultValue="next"
+    >
+      Specifies the type of nav button to include. Options include <stache-code>finish</stache-code>, <stache-code>next</stache-code>, and <stache-code>previous</stache-code>.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="disabled"
+      isOptional="true"
+      defaultValue="false"
+    >
+      Indicates whether to disable the nav button.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="progressIndicator"
+    >
+      Specifies the progress indicator component to associate with the nav button.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-example>
+    <div class="sky-demo-bg-gray">
```

Comment:
```suggestion
    <div>
```

Reply:
Changing this should resolve Adam's comment

Reply:
I'll remove that background color. Sorry about that.